docs/config/lua/config/default_cwd.md

```diff
@@ -0,0 +1,4 @@
+# `default_cwd`
+
+Specifies the default current working directory for the initially spawned
+program.
```

Comment:
Would be great to expand on this because the outcome is a bit more complicated/nuanced; it's probably a good idea to explain this by adding a `Current Working Directory` section to https://wezfurlong.org/wezterm/config/launch.html#launching-programs, as well as covering the same information here:

If `wezterm start --cwd /some/path` is used to specify the current working directory, that will take precedence for the initially created window.

Otherwise, the initial window will use the value specified by the `default_cwd` configuration option.

Commands launched using `SpawnCommand` (link to https://wezfurlong.org/wezterm/config/lua/SpawnCommand.html) will use the `cwd` specified in the `SpawnCommand` (if any).

Otherwise: Panes/Tabs/Windows created after the first will generally try to resolve the current working directory of the current Pane, preferring a value set by OSC 7 (link to Shell Integration: https://wezfurlong.org/wezterm/shell-integration.html) and falling back to attempting to lookup the cwd of the current process group leader attached to a local Pane.  If no cwd can be resolved, then the `default_cwd` will be used.  If `default_cwd` is not specified, then the home directory of the user will be used.